ci-operator/step-registry/openshift/image-based/upgrade/recipient/gather/lca/openshift-image-based-upgrade-recipient-gather-lca-commands.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -xeuo pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+remote_workdir=$(cat ${SHARED_DIR}/remote_workdir)
+instance_ip=$(cat ${SHARED_DIR}/public_address)
+host=$(cat ${SHARED_DIR}/ssh_user)
+ssh_host_ip="$host@$instance_ip"
+RECIPIENT_VM_NAME=$(cat ${SHARED_DIR}/recipient_vm_name)
+recipient_kubeconfig=${remote_workdir}/ib-orchestrate-vm/bip-orchestrate-vm/workdir-${RECIPIENT_VM_NAME}/auth/kubeconfig
+
+echo "Using Host $instance_ip"
+
+SSHOPTS=(-o 'ConnectTimeout=5'
+  -o 'StrictHostKeyChecking=no'
+  -o 'UserKnownHostsFile=/dev/null'
+  -o 'ServerAliveInterval=90'
+  -o LogLevel=ERROR
+  -i "${CLUSTER_PROFILE_DIR}/ssh-privatekey")
+
+cat <<EOF > ${SHARED_DIR}/gather_recipient_lca.sh
+#!/bin/bash
+set -euo pipefail
+
+cd ${remote_workdir}
+oc --kubeconfig ${recipient_kubeconfig} adm inspect ns/openshift-lifecycle-agent --dest-dir=./must-gather-lca-${RECIPIENT_VM_NAME}
+
```

Comment:
Seems that this script is executed on the recipient, if that's the case, can we get the output of:
```
journalctl -u installation-configuration.service 
cat /var/tmp/recert-summary.yaml
```

Reply:
@eranco74 this has been added